apps/web/src/pages/InstructorLedTrainingPage/InstructorLedTrainingPage.tsx

```diff
@@ -0,0 +1,545 @@
+                    <Button
+                      onClick={() =>
+                        setTrainingOpportunitiesFilteredBy(
+                          CourseLanguage.French,
+                        )
+                      }
+                      {...(trainingOpportunitiesFilteredBy ===
+                      CourseLanguage.French
+                        ? selectedFilterStyle
+                        : unselectedFilterStyle)}
+                    >
+                      {intl.formatMessage({
+                        defaultMessage: "French only",
+                        id: "wBU9X4",
+                        description:
+                          "Filter by option on instructor training page.",
+                      })}
+                    </Button>
+                  </div>
+                  {trainingOpportunitiesState.map((trainingOpportunity) => {
+                    const localizedTitle = getLocalizedName(
+                      trainingOpportunity.title,
+                      intl,
+                    );
+                    return (
+                      <CardBasic
+                        key={trainingOpportunity.id}
+                        data-h2-margin-bottom="base(x1)"
+                      >
+                        <div
+                          data-h2-display="base(flex)"
+                          data-h2-flex-direction="base(column) l-tablet(row)"
+                          data-h2-justify-content="base(center) l-tablet(flex-start)"
+                          data-h2-align-items="base(center)"
+                          data-h2-gap="base(x.75) l-tablet(x.5)"
+                          data-h2-margin-bottom="base(x.5)"
+                        >
+                          <Heading
+                            Icon={CalendarIcon}
+                            level="h4"
+                            size="h5"
+                            data-h2-font-weight="base(700)"
+                            data-h2-margin="base(0)"
+                            data-h2-text-align="base(center) l-tablet(initial)"
+                          >
+                            {localizedTitle}
+                          </Heading>
+                          <CourseLanguageChip
+                            courseLanguage={
+                              trainingOpportunity.courseLanguage?.value
+                            }
+                          />
+                        </div>
+                        <MetaData
+                          metaData={getMetaData(intl, trainingOpportunity)}
+                          data-h2-align-items="base(center)"
+                        />
+                        <CardSeparator />
+                        <Dialog.Root>
+                          <Dialog.Trigger>
+                            <Button mode="text" data-h2-font-weight="base(700)">
+                              {intl.formatMessage({
+                                defaultMessage: "Learn more and apply",
```

Comment:
This seems like an a11y problem to me.  I wonder if it should be something like `Learn more and apply<hidden> to course name</hidden>`?

Reply:
I'll ask @NienkeBr and see what she thinks?

Reply:
![image](https://github.com/user-attachments/assets/d50ecaa1-a4bd-4d04-b8c7-66ff9c9e1b75)

Nice catch Peter, fixed in [1d5d7e2](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12083/commits/1d5d7e2f60b677fe827c98dcfa6e8b11256d0b8d)